extension/commands/openTransactionViewCommand.ts

```diff
@@ -0,0 +1,70 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+*/
+'use strict'; 
+import * as vscode from 'vscode';
+import { IBlockchainQuickPickItem, UserInputUtil } from './UserInputUtil';
+import { FabricConnectionManager } from '../fabric/FabricConnectionManager';
+import { VSCodeBlockchainOutputAdapter } from '../logging/VSCodeBlockchainOutputAdapter';
+import { LogType } from '../logging/OutputAdapter';
+import { ExtensionCommands } from '../../ExtensionCommands';
+import { InstantiatedTreeItem } from '../explorer/model/InstantiatedTreeItem';
+import { FabricChaincode } from '../fabric/FabricChaincode';
+import { TransactionView } from '../webview/TransactionView';
+
+export async function openTransactionView(context: any, treeItem?: InstantiatedTreeItem): Promise<void> {
```

Comment:
`context: vscode.ExtensionContext`

Reply:
you also don't need to pass in the context you can get it from the global state